contents.js

```diff
@@ -0,0 +1,504 @@
+exports.namesList = [
```

Comment:
change the name of the file please. something like namesDictionary.js

Reply:
It does not only include names though

Reply:
name has been changed